text/0000-floating-point-additional-precision.md

```diff
@@ -48,6 +48,14 @@ will never make any floating-point operation *less* accurate, but it can make
 floating-point operations *more* accurate, making the result closer to the
 mathematically exact answer.
 
+Due to differences in hardware, in platform libm implementations, and various
+other factors, Rust cannot fully guarantee identical results on all target
+platforms. (Doing so on *all* platforms would incur a massive performance
+loss.) However, with some additional care, applications desiring cross-platform
+identical results can potentially achieve that on multiple target platforms. In
+particular, applications prioritizing identical, portable results across two or
+more target platforms can disable extra floating-point precision entirely.
```

Comment:
As I mentioned [in a previous comment](https://github.com/rust-lang/rfcs/pull/2686#issuecomment-486197979), mere reproducibility is not always the reason to disable this behaviour. Some algorithms can actually take advantage of the ~weird~ special properties of floating-point arithmetic. Such algorithms should remain implementable as Rust libraries, and those should not break just because someone decided they wanted *their* unrelated floating-point code to be as fast as possible.